### DISCUSSION TITLE
Voting system: let's keep the ball rolling!

### DISCUSSION THREAD

**Discussion by deluan** [opensubsonic/open-subsonic-api#40] [General]
Alright, so we have received several requests, which is great. Now, we need to determine the voting system and organization to begin working on these requests.

Regarding the voting system, let’s aim for clarity and transparency. I propose the following approach:

Use reactions to cast votes as they are easy to count and identify the voters:

:+1: will indicate a positive vote.
:-1: will indicate a negative vote.

The votes should be made on the first post of each requests.

Regarding the voting details, I suggest we focus on establishing two “rules”:

In the case of only positive votes:

- How many servers and clients should be involved in the validation process?

If there is at least one negative vote:

- Should a negative vote from a server hold more significance than a vote from a client? How can we address this distinction?
- Can we make adjustments or modifications until we eliminate all negative votes?
- Should we consider a rule where each positive vote cancels out one negative vote (e.g., X +1 vote erases 1 negative vote)?
- Are there any alternative approaches that we should consider in this context?

Let’s discuss these points and any other pertinent matters related to the voting system and validation process. Your input and suggestions are valuable in finalizing these aspects.

Additionally, let’s open the floor for further discussion on any other aspects or alternative approaches to the voting and validation process.

Please share your thoughts and suggestions to refine this voting system.

**Comment by deluan** [comment]
@opensubsonic/servers @opensubsonic/clients

**Comment by Tolriq** [comment]
1) I would say 2 server and 1 client is enough for validation

2) I would say that negative votes without argumentation or counter proposal just count as -1 and is not blocking. If there's argumentation or counter proposal then we should aim to try to reach a consensus. If it's not possible then we should aim to have more servers validating the feature to compensate.

**Remember that OpenSubsonic is all about nearly everything being optional, if a server does not want or can't support a feature it just don't. No need to prevent others from supporting it.**

**Comment by Tolriq** [comment]
So not a lot of discussions around this, I guess it's a go with current proposal.

Be ready for September, I'll try to allocate more time to this and start making the actual API proposals to vote on.
